### PR TITLE
Update Zen Account on RPI Login

### PR DESCRIPTION
**Pull request by Jeddf**
- `/rpi/edit` route for editing rpi profile
- Zen profile update on rpi login with known account

Partially resolves issue https://github.com/CoderDojo/cp-zen-platform/issues/1387
returnTo on profile edit is dependent on https://github.com/RaspberryPiFoundation/profile/pull/1027

### CODE REVIEW

File: web/lib/rpi-auth.js [hunk 1]
```diff
@@ -42,11 +43,18 @@ function getLogoutRedirectUri() {
 
 function getRegisterRedirectUri() {
   const params = new URLSearchParams();
-  params.set('brand', 'coderdojo');
+  params.set('brand', brand);
```
Comment (grega) [inline web/lib/rpi-auth.js]: I might well be missing something, but where is `brand` being set?

File: web/lib/rpi-auth.js [hunk 2]
```diff
@@ -42,11 +43,18 @@ function getLogoutRedirectUri() {
...
   params.set('returnTo', `${homeServer}${loginPath}`);
   return `${profileServer}${profileSignupPath}?${params}`;
 }
 
+function getEditRedirectUri() {
+  const params = new URLSearchParams();
+  params.set('brand', brand);
```
Comment (grega) [inline web/lib/rpi-auth.js]: Same as above.

File: web/lib/rpi-auth.js (hunk 1 above)
Comment (Jeddf) [reply web/lib/rpi-auth.js]: It's just a string const being set up on line 18

File: web/lib/rpi-auth.js (hunk 2 above)
Comment (Jeddf) [reply web/lib/rpi-auth.js]: Again up on L18, I'm gonna clean up that block of variables up there, I've left 'em very messy!

File: web/lib/rpi-auth.js (hunk 1 above)
Comment (grega) [reply web/lib/rpi-auth.js]: Ha, of course - I should really look at the whole file rather than the diff :)

**Pull request merged**